src/qt/radiobox.cpp

```diff
@@ -242,6 +245,9 @@ bool wxRadioBox::Show(unsigned int n, bool show)
 
 bool wxRadioBox::Show( bool show )
 {
+    if (!m_qtGroupBox)
```

Comment:
This is not quite right, `Show()` can be called before creating the window, see `wxWindowMSW::Show()` in `src/msw/window.cpp`. So we should call the base class version first instead of always returning `false` in this case.